### ISSUE TITLE
Add text-to-speech to the TeamTalk Mac client

### ISSUE THREAD

**Issue by slannon97** [BearWare/TeamTalk5#450]
TeamTalk 5 for macOS does not work very well with the VoiceOver screen reader. It is usable if you use standard macOS keyboard commands such as tab and the arrow keys, but the most significant issues are inaccessible lists. I understand VoiceOver and QT applications do not work that well together, so I am suggesting the use of text-to-speech to address these accessibility issues.

The Mac client does not currently support text-to-speech at all. Essentially, the text-to-speech would use the default system voice on the Mac to both speak server messages as it does on Windows with TeamTalk 5 Classic, as well as automatically reading the contents of lists when using the arrow keys to navigate. This would make TeamTalk for VoiceOver users completely accessible without requiring a complete rewrite of the app.

**Comment by bear101** [comment]
Qt Text-to-Speech started on this branch: https://github.com/BearWare/TeamTalk5/tree/QtTextToSpeech

**Comment by bear101** [comment]
Duplicate #275
